### ISSUE TITLE
Add more information/parameters to callbacks

### ISSUE THREAD

**Issue by Quurks** [felixhagspiel/jsOnlyLightbox#33]
The current callbacks are nice, but they are - as far as I can tell - lacking important information.
I can see no way to get the actual image which was opened, or which image failed to load.

It would be nice to have more information in the event objects / event objects at all.

**Comment by felixhagspiel** [comment]
That is true. As I am pretty busy atm, it might take a while until I can implement it. But you are also welcome to contribute if you got time :)
